package.json

```diff
@@ -31,7 +31,6 @@
     "@typescript-eslint/parser": "^4.10.0",
     "array-shuffle": "^2.0.0",
     "await-sleep": "0.0.1",
-    "buffer": "^6.0.3",
```

Comment:
@harbu heh I only just added this.

Problem was that the version of buffer that webpack 4 bundled was very outdated https://github.com/streamr-dev/streamr-client-protocol-js/commit/b749a487cc33afbf0bad6a6cba2245d0165c8794

hence this alias:
```js
    resolve: {
        alias: {
            buffer: path.resolve(__dirname, 'node_modules', 'buffer'),
        }
    },
```
 
Probably should have added a comment near that alias or more description in the commit.